### ISSUE TITLE
The documentation for the npm command line is wrong

### ISSUE THREAD

**Issue by dong-lufei** [npm/cli#6066]
```
npm -l
.
.
.
login           Add a registry user account

                Usage:
                npm adduser

                Options:
                [--registry <registry>] [--scope <@scope>]
                [--auth-type <legacy|web|sso|saml|oauth|webauthn>]

                aliases: login, add-user

                Run "npm help adduser" for more info
```
The column menu login on the left of the detailed list should be adduser, or change the grammar and aliases inside

**Comment by lukekarrys** [comment]
What version of npm is this on?

**Comment by dong-lufei** [comment]
> 这是什么版本的 npm？

The latest stable version of nodejs I installed contains npm version 8.19.3

**Comment by Hafizur046** [comment]
login and adduser were separated in #5550 which was released in v9.0.0. Before that, the login command was just an alias for adduser.

**Comment by lukekarrys** [comment]
yes, this has been changed in npm v9. Closing as this is no longer an issue.
